docs/src/Examples/Demo/BasicDateTimePicker.jsx

```diff
@@ -44,6 +44,17 @@ export default class BasicDateTimePicker extends PureComponent {
             showTodayButton
           />
         </div>
+
+        <div className="picker">
+          <DateTimePicker
```

Comment:
Please remove additional examples, just change the logic

Reply:
I added the additional examples to have examples which use the today picker + autoOk since there were no autoOk examples yet, but i can remove them :)

Reply:
I think prop description for `autoOk` is sufficient

Reply:
okay, then i will adapt the pr :)